TWLight/templates/header_partial_b4.html

```diff
@@ -104,6 +104,9 @@
                 {% trans "Log out" %}
               </a>
             </li>
+          {% elif not user.is_authenticated %}
+            <li role="presentation" class="mobile-menu align-middle nav-item">
+              <a class="btn twl-btn" href="{% url 'oauth_login' %}?next={{request.path}}">
```

Comment:
Please add a `trans` Django element with `Log In` as its text. See the Log out button as an example.